### ISSUE TITLE
Abstraction of status changes

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/john-travoltage#171]
It looks like an element ID (peerID) is passed to components like ElectronLayerNode, where:

1. The component grabs a corresponding HTMLElement with ```document.getElementById( peerID )```
2. If that exists, updates status by first setting its textContent to an empty string ```''```, then to the actual string.

This would be much cleaner if components were instead passed a function like:
```js
function statusChangeCallback( {string} status )
```
thus components don't have to care about the DOM (and particularly, whether the DOM element exists).

Code review for https://github.com/phetsims/john-travoltage/issues/128

**Comment by jessegreenberg** [comment]
Related to #134.

**Comment by jonathanolson** [comment]
Also applies to alerts.

**Comment by jessegreenberg** [comment]
@jonathanolson the type that now handles status and alert changes is called [AriaHerald](https://github.com/phetsims/balloons-and-static-electricity/blob/master/js/balloons-and-static-electricity/accessibility/AriaHerald.js) and I would like to move it to common code.  Do you have any recommendation on where it should exist?

My thoughts are that it can't be scenery (it is specific to PhET sims). It shouldn't be phetcommon since that repo is intended to be deprecated (https://github.com/phetsims/phetcommon/issues/24), and it can't be phet-core because it is now using axon/Property for PhET-iO (it looks like phet-core doesn't have dependencies).

**Comment by jonathanolson** [comment]
If it cant be scenery (because it's truly phet-specific, more than just a Tandem dependency), then scenery-phet would probably be the place.

**Comment by jessegreenberg** [comment]
Thanks, I will proceed with scenery-phet.  It is phet-specific because it uses HTML that must exist in our HTML templates, and the attributes on the HTML used in the templates were chosen specifically for the context of PhET sims.

**Comment by jessegreenberg** [comment]
Alerts and status updates have been abstracted with [AriaHerald](https://github.com/phetsims/scenery-phet/blob/master/js/accessibility/AriaHerald.js), which this sim now uses.

This static object doesn't use callbacks, but allows setting of alerts/status updated in an abstracted way.  It is used in this sim in ElectronLayerNode, SparkNode, and AppendageNode.
```js
AriaHerald.announcePoliteWithStatus( StringUtils.format( chargeText, currentCharge, priorCharge ), true );
```

Does this seem acceptable @jonathanolson?

**Comment by jessegreenberg** [comment]
Oops, incorrect assignee, apologizes @jobara.

**Comment by jessegreenberg** [comment]
This issue is stale, utteranceQueue now manages all alerts.
